backends/cuda-shared/ceed-cuda-shared-basis.c

```diff
@@ -271,27 +315,34 @@ int CeedBasisCreateTensorH1_Cuda_shared(CeedInt dim, CeedInt P_1d, CeedInt Q_1d,
   ierr = CeedBasisGetNumComponents(basis, &num_comp); CeedChkBackend(ierr);
   char *basis_kernel_path, *basis_kernel_source;
   ierr = CeedGetJitAbsolutePath(ceed,
-                                "ceed/jit-source/cuda/cuda-shared-basis.h",
+                                "ceed/jit-source/cuda/cuda-shared-basis-tensor.h",
                                 &basis_kernel_path); CeedChkBackend(ierr);
   CeedDebug256(ceed, 2, "----- Loading Basis Kernel Source -----\n");
   ierr = CeedLoadSourceToBuffer(ceed, basis_kernel_path, &basis_kernel_source);
   CeedChkBackend(ierr);
   CeedDebug256(ceed, 2, "----- Loading Basis Kernel Source Complete -----\n");
-  ierr = CeedCompileCuda(ceed, basis_kernel_source, &data->module, 8,
+  ierr = CeedCompileCuda(ceed, basis_kernel_source, &data->module, 9,
                          "BASIS_Q_1D", Q_1d,
                          "BASIS_P_1D", P_1d,
-                         "BASIS_T_1D", CeedIntMax(Q_1d, P_1d),
+                         "T_1D", CeedIntMax(Q_1d, P_1d),
                          "BASIS_BUF_LEN", num_comp * CeedIntPow(Q_1d > P_1d ?
```

Comment:
I went ahead and removed `BASIS_BUF_LEN` from compile-time definitions in hip-shared -- we don't use it any more in the kernels